configs/11.0/packages/gcc/sources

```diff
@@ -32,9 +32,11 @@ ATSRC_PACKAGE_RELFIXES=
 ATSRC_PACKAGE_NAMESUFFIX="[C, C++ (g++), fortran, Go]"
 ATSRC_PACKAGE_STR_VER="${ATSRC_PACKAGE_NAME} ${ATSRC_PACKAGE_VER}-ibm-r${ATSRC_PACKAGE_REV}"
 ATSRC_PACKAGE_PRE="test -d gcc-${ATSRC_PACKAGE_VER}-${ATSRC_PACKAGE_REV}"
-ATSRC_PACKAGE_CO=([0]="svn co --revision ${ATSRC_PACKAGE_REV} svn://gcc.gnu.org/svn/gcc/branches/ibm/gcc-7-branch" \
-		  [1]="svn co --revision ${ATSRC_PACKAGE_REV} http://gcc.gnu.org/svn/gcc/branches/ibm/gcc-7-branch")
-ATSRC_PACKAGE_POST="mv gcc-7-branch gcc-${ATSRC_PACKAGE_VER}-${ATSRC_PACKAGE_REV}"
+ATSRC_PACKAGE_CO=([0]="git clone git://gcc.gnu.org/git/gcc.git")
```

Comment:
You could still implement this part with just 2 commands (git-clone + git-checkout):
```
git clone -c remote.origin.fetch="+refs/vendors/ibm/heads/*:refs/remotes/origin/ibm/*" git://gcc.gnu.org/git/gcc.git
git checkout -b gcc-${ATSRC_PACKAGE_VER}${ATSRC_PACKAGE_REV:+-${ATSRC_PACKAGE_REV}} ${ATSRC_PACKAGE_REV}
```
Anyway, your implementation looks good to me.

Reply:
> You could still implement this part with just 2 commands (git-clone + git-checkout):

This works with relatively new git. In older git, there was an issue with -c option of clone, especially when setting remote.origin.fetch. That's why I've implemented this part with 3 commands.